fury/data/fetcher.py

```diff
@@ -24,6 +24,9 @@
 UW_RW_URL = \
     "https://digital.lib.washington.edu/researchworks/bitstream/handle/"
 
+NEW_ICONS_DATA_URL = \
+    "https://raw.githubusercontent.com/ganimtron-10/fury-data/master/icons/new_icons/"
```

Comment:
Once this [PR](https://github.com/fury-gl/fury-data/pull/9) gets merged I will update the link with the official fury-data repo.